source/config/__init__.py

```diff
@@ -19,6 +25,18 @@
 import easeOfAccess
 import winKernel
 
+#Choices for numbers combo box in Voice settings.
+
+digitChoices=[
+	# Translators: Choice in a combo box for speaking numbers by word.
+	"Words",
+	# Translators: Choice in a combo box for speaking numbers as digits.
+	"Digits",
+	# Translators: Choice in a combo box for speaking numbers as double digits.
+	"Double Digits",
+	# Translators: Choice in a combo box for speaking numbers as triple digits.
+	"Triple Digits"]
+
 def validateConfig(configObj,validator,validationResult=None,keyList=None):
 	"""
 	@deprecated: Add-ons which need this should provide their own implementation.

```

Comment:
I can't figure out why, but config doesn't seem to have _ available 
raising a name error. when and where does NVDA create _? I looked in 
several places and couldn't find it.


On 5/6/2016 6:04 AM, James Teh wrote:
>
> In source/config/__init__.py 
> <https://github.com/nvaccess/nvda/pull/5930#discussion_r62321173>:
>
> > @@ -19,6 +25,18 @@
> >  import easeOfAccess
> >  import winKernel
> >
> > +#Choices for numbers combo box in Voice settings.
> > +
> > +digitChoices=[
> > +	# Translators: Choice in a combo box for speaking numbers by word.
> > +	"Words",
> > +	# Translators: Choice in a combo box for speaking numbers as digits.
> > +	"Digits",
> > +	# Translators: Choice in a combo box for speaking numbers as double digits.
> > +	"Double Digits",
> > +	# Translators: Choice in a combo box for speaking numbers as triple digits.
> > +	"Triple Digits"]
> > +
> >  def validateConfig(configObj,validator,validationResult=None,keyList=None):
> >  	"""
> >  	@deprecated: Add-ons which need this should provide their own implementation.
> I assume you mean remove it; it's already deprecated. Sure, but please 
> submit a separate PR for that. Btw, the strings you intend to be 
> translatable aren't translatable. You need to enclose translatable 
> strings in _(); e.g. _("Words").
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/5930/files/7507485b589cdf070e32b05a0a87e4de96d812af#r62321173>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194